tests/test_ace.py

```diff
@@ -6,8 +6,8 @@
 
 import pyne.ace
 
-
-def setup():
+@pytest.fixture(autouse=True)
+def setup_function():
```

Comment:
I think `setup_module()` will recover the intended behavior and don't use the fixture:
```suggestion
def setup_module():
```